src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -586,7 +586,22 @@ private void DynamoView_Loaded(object sender, EventArgs e)
                 Converter = new BooleanToVisibilityConverter()
             };
             BackgroundPreview.SetBinding(VisibilityProperty, vizBinding);
-            Analytics.TrackTimedEvent(Categories.Performance, "ViewActivation", dynamoViewModel.Model.stopwatch.Elapsed, "DynamoModel + DynamoView loaded");
+            TrackStartupAnalytics();
+        }
+
+        private void TrackStartupAnalytics()
+        {
+            if (!Analytics.ReportingAnalytics) return;
+
+            string packages = string.Empty;
+            var pkgExtension = dynamoViewModel.Model.GetPackageManagerExtension();
+            if(pkgExtension != null)
+            {
+                packages = pkgExtension.PackageLoader.LocalPackages
+                    .Select(p => p.Name)
+                    .Aggregate((x, y) => string.Format("{0}, {1}", x, y));
+            }
+            Analytics.TrackTimedEvent(Categories.Performance, "ViewStartup", dynamoViewModel.Model.stopwatch.Elapsed, packages);
```

Comment:
This can contain names, and so needs to be sent to instrumentation not analytics